src/redisraft.c

```diff
@@ -555,6 +555,30 @@ static void handleAsking(RedisRaftCtx *rr, RedisModuleCtx *ctx)
     RedisModule_ReplyWithSimpleString(ctx, "OK");
 }
 
+static void handleEndSession(RedisRaftCtx *rr, RedisModuleCtx *ctx, char *reason)
+{
+    RaftReq *req = RaftReqInit(ctx, RR_END_SESSION);
+    if (req == NULL) {
```

Comment:
It won't be null ever. We can skip the check

Reply:
well, this code is about to be rewritten, req can't be created here (no req on disconnect, this actually was the problem I hit earlier that I thought was a redis bug, still might be, but I was doing something bad.  So will have to have a req ==/!= NULL code path, but this isn't it.